enclave-tls/samples/enclave-tls-client/client.c

```diff
@@ -39,19 +43,47 @@ static sgx_enclave_id_t load_enclave(bool debug_enclave)
 	int ret = sgx_create_enclave(ENCLAVE_FILENAME, debug_enclave, &t, &updated, &eid, NULL);
 	if (ret != SGX_SUCCESS) {
 		ETLS_ERR("Failed to load enclave %d\n", ret);
-		return -1;
+		return 0;
 	}
 
-	ETLS_DEBUG("Success to load enclave id %ld\n", eid);
+	ETLS_INFO("Success to load enclave id %ld\n", eid);
```

Comment:
Change to `Success to load enclave with enclave id %ld\n", eid);`

Reply:
Plz fix this.

Reply:
Plz fix this.

Reply:
ok. sorry, miss this